client/actions/appointmentAction.js

```diff
@@ -45,9 +43,7 @@ export function getBookingStatus(email) {
             .then((response) => {
                 dispatch(getBookingStatusAction(response))
             })
-            .catch((err) => {
-                console.log(err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}